pypeit/core/wavecal/defs.py

```diff
@@ -29,6 +29,39 @@ def instruments():
     return instr_dict
 
 
+class LinesBitMask(BitMask):
+    """
+    Bits for arc lines
+    """
+    version = '1.0.0'
+
+    def __init__(self):
+        mask = dict([
+            ('ArI', 'Argon I'),
+            ('HgI', 'Comment'),
+            ('KrI', 'Comment'),
+            ('NeI', 'Comment'),
+            ('XeI', 'Comment'),
+            ('CdI', 'Comment'),
+            ('ZnI', 'Comment'),
+            ('HeI', 'Comment'),
+            ('OH_R24000', 'Comment'),
+            ('OH_triplespec', 'Comment'),
+            ('CuI', 'Comment'),
+            ('ArII', 'Comment'),
+            ('OH_XSHOOTER', 'Comment'),
+            ('OH_GNIRS', 'Comment'),
+            ('OH_NIRES', 'Comment'),
+            ('ThAr_XSHOOTER_VIS', 'Comment'),
+            ('OH_GMOS', 'Comment'),
+            ('OH_MODS', 'Comment'),
+            ('ThAr_MagE', 'Comment'),
+            ('OH_FIRE_Echelle', 'Comment'),
+            ('Ar_IR_GNIRS', 'Comment'),
```

Comment:
I noticed that `UNKNWNs.dat` is not included here. Are we removing this list?

(We are also missing `FeI`, but I've added that to my `KCWI6` branch, as it shouldn't affect anyone else).

Reply:
Said differently, I think line 271 of `wv_fitting.py`:
`all_idsion = np.array(['UNKNWN']*len(tcent))`
requires `UNKNWN` to be included in the bitmask?

Reply:
added.